src/Packagist/WebBundle/Resources/config/services.yml

```diff
@@ -26,14 +26,12 @@ services:
 
     packagist.user_repository:
         class: Packagist\WebBundle\Entity\UserRepository
-        factory_service: doctrine
-        factory_method: getRepository
+        factory: [@doctrine, getRepository]
```

Comment:
Why? This is needed only for string that does not represent a service AFAIK. Am I wrong?